src/main/java/org/opensearch/security/securityconf/impl/v7/RoleV7.java

```diff
@@ -29,6 +29,7 @@
 
 import java.util.ArrayList;
 import java.util.Collections;
+import java.util.Date;
```

Comment:
- This is a first in this class, is this a version of https://github.com/opensearch-project/security/issues/3079 that you wanted to include for the new type?

Reply:
To be honest, I just thought it seemed like it may be a nice thing to have since we were adding the new class. I can also remove it though and don't feel strongly about its inclusion.